source/settings/plugin/sql-cassandra.rst

```diff
@@ -0,0 +1,335 @@
+.. _sql-cassandra:
+
+====================
+SQL Driver Cassandra
+====================
+
+Driver name ``cassandra``.
+
+Driver for Apache Cassandra CQL server.
+
+To compile support for this driver, you need to have DataStax C/C++ driver and headers installed.
+
+Supported options
+=================
+
+``host``
+--------
+- Default: <empty>
+- Values: :ref:`string`
```

Comment:
Should these be converted to the new settings formats?

Reply:
Should.